packages/nextjs/generated/deployedContracts.ts

```diff
@@ -1,3 +1,15 @@
-const deployedContracts = null;
+import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract";
+
+/**
+ * @example
+ * const deployedContracts = {
+ *   31337: {
+ *    YourContract: {
+ *      address: "0x..."
+ *      abi: [...],
+ *    }
+ * } satisfies GenericContractsDeclaration | null;
+ */
+const deployedContracts = null satisfies GenericContractsDeclaration | null;
 
 export default deployedContracts;
```

Comment:
We are using `deployedContracts` here but `contracts` from `99_generateTsAbis.ts`.

I like deployedContracts (More on the naming later)